tests/testing_tests/test_estimators.py

```diff
@@ -77,7 +79,7 @@ class TestLogisticRegressionEstimator(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls) -> None:
-        cls.scarf_df = pd.read_csv("tests/data/scarf_data.csv")
+        cls.scarf_df = pd.read_csv("tests/resources/data/scarf_data.csv")
```

Comment:
@jmafoster1 Why did this change to `test/resources/data`? Do we have other subfolders in `tests/resources`?

Reply:
There was a folder called `tests/data` and one called `tests/resources/data`, both containing test data, so I thought I'd condense everything into one folder.